spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-describe ManageIQ::Providers::IbmCloud::VPC::CloudManager::Refresher, :vcr => {:allow_playback_repeats => true} do
```

Comment:
Switching to `VCR.use_cassette` so that we can re-use the full refresh cassette before the targeted refresh